### ISSUE TITLE
encoding of directives (protective texts)

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#2249]
I recently worked with a few protective prayers in Geez, which all ended with an Amharic directive regarding their execution. I think it could make sense to create a new value for `@type` ("directive") for `<explicit>` for such cases, to be able to encode them well in the manuscript description. What do you think?
For example LIT6706PPSalotaHabt
@karljonaskarlsson @MassimoVilla @DenisNosnitsin1970

**Comment by DenisNosnitsin1970** [comment]
Well, "directive" and "explicit" are two different levels of text (semantics against spacial text division ?),  I do not think they should be treated equally, as I see in the record. Please note also that we have "Directive" as Addition type, where it is primarily understood as directives (concerning the liturgical use). It must not be like this, but some kind of limitation is necessary because "directive" as imperative/jussive can be encountered everywhere.

**Comment by thea-m** [comment]
Yes, it is a different type of text from the "directive" addition type. 
Could you suggest another way of encoding it then? I think the explicit solution is practical and easy, but if you find another that fits better, that would of course be nice.

**Comment by karljonaskarlsson** [comment]
Does it have to be "incipit" or "explicit"? Couldn't it just be encoded as an excerpt of text? (Because if I remember correctly, they are not the end of the text, which is what I would normally understand as the "explicit")

**Comment by thea-m** [comment]
It can, but then I don't think it would be possible to search and filter for this.

**Comment by thea-m** [comment]
I think we may not understand each other completely. The texts I mean are Geez protective prayers. Each prayer ends with a short Amharic section, part of the main body of the text, that (as far as I understand it, not everything is clear to me) gives directives to carry out actions to achieve the aim of the prayer. As "explicit" of the work, I would understand the last part of the Geez prayer, before final supplications. 
The directive is its own section. In the work record, this is simple to encode ( ` <div type="textpart" subtype="directive" xml:lang="am" xml:id="ExplicitDirective">`). In the manuscript, this can of course be referred to as an msItem using LIT1234XXX#ExplicitDirective. 
My idea was that, similarly to the encoding of supplications and subscriptions, the manuscript descriptions would be more reader-friendly if these short directives were directly encoded within the msItem.

**Comment by DenisNosnitsin1970** [comment]
This is a bit complicated and without a concrete example not transparent. If I understand you right, introducing   `<div type="textpart" subtype="directive" xml:lang="am" xml:id="ExplicitDirective">` into the work record, we still move towards a different level (trying to select semantic categories within the physical end of the work?
), as explicit is normally understood in the sense of termination of the work [http://www.palaeographia.org/vocabulaire/pages/vocab2.htm](url), balanced by the same for incipit...

**Comment by DenisNosnitsin1970** [comment]
I see, it is quite complex. If you ask me, roughly, I see in the work the spacial divisions: incipit, explicit, body of the text that can be divided into chapters. Within incipit or explicit, there may be parts of text parts (or incipit/explicit parts) that can be marked as "title", "supplication", "directive", and possible more.  If I understand right, yes, to a certain degree,   `explicit type="supplication"` makes sense, I would rather think of making the same for directive, but not of exchanging "explicit" for "directive".

**Comment by thea-m** [comment]
I don't understand well what is meant by "but not of exchanging "explicit" for "directive"". I was suggesting to encode the directive in addition to the explicit proper of the text. Is that what you mean?
